src/microfrontend/useAppAssets.ts

```diff
@@ -48,24 +48,18 @@ const useAppAssets = (appName: string, staticPaths: string[] = [], pathToManifes
     );
 
     useEffect(() => {
-        const loadAppFromManifest = async (pathToManifest: string) => {
+        const loadAssets = async (pathToManifest: string | undefined, staticPaths: string[]) => {
             try {
-                const manifest = await fetchAssetManifest(createAssetManifestUrl(pathToManifest));
-                const pathsToLoad = extractPathsToLoadFromManifest(manifest);
-
-                await loadjs(pathsToLoad, appName, {
-                    returnPromise: true,
-                });
-
-                setStatus(AssetStatus.Klar);
-            } catch (e) {
-                setStatus(AssetStatus.Feil);
-            }
-        };
-
-        const loadDefinedAssets = async (staticPaths: string[]) => {
-            try {
-                await loadjs(staticPaths, appName, {
+                let liste: string[] = staticPaths;
```

Comment:
Rename denne til noe bedre? Hva med å kalle denne for `pathsToLoad` og rename den på L58 til `pathsFromManifest`?